### PR TITLE
Document do-not-delete annotation in 2.6

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/24036

### CODE REVIEW

**Comment by chenz4027** [comment]
@oafischer Is it ready for review or you are still working on it? Saw some commits lately

**Comment by oafischer** [comment]
@chenz4027 Wow you're fast! It's ready for review, typos and stuff are inevitable so I fix those whenever I see them. When I ask for dev review I'm mainly concerned about technical accuracy, we catch the writing stuff during peer review (usually after I get dev approval to make sure the content is close to its final form).

**Comment by chenz4027** [comment]
@oafischer haha np, just browsing my email lists and found a notification ;) will review it now

File: applications/manage_apps_git.adoc [hunk 1]
```diff
@@ -92,4 +92,31 @@ Your directory structure is created for the following subscription flow: `root-s
...
+* See policy examples in https://github.com/stolostron/policy-collection[Policy collection].
+
+[#git-keep-resource-after-sub-delete]
+== Keeping deployed resources after deleting subscription with Git
+
+When creating subscriptions using a Git repository, you can add a `do-not-delete` annotation to keep specific deployed resources after you delete the subscription. The `do-not-delete` annotation only works with top level deployment resources. To add the `do-not-delete` annotation, complete the following steps:
+
+. Create a subscription that deploys at least one resource.
+
+. Add the following annotation to the resource or resources you want to keep when deleting the subscription:
```
Comment (chenz4027) [inline applications/manage_apps_git.adoc]: I'm okay with this description. But the user could also put the label in the app's original yaml file so they won't need to add it on delete. i think it's no biggie if we don't mention it and the description is pretty clear.

**Review by chenz4027 — APPROVED** [review]
/hold
/lgtm
Added one comment, up to @oafischer whether to add or keep it as it is. I'm good either way. Feel free to unhold it once you are ready to merge

File: applications/manage_apps_git.adoc (hunk 1 above)
Comment (oafischer) [reply applications/manage_apps_git.adoc]: I think my writing may be ambiguous, what I'm trying to say is to add the annotation for resources you want to keep when, in the future, you might decide to delete the subscription. Thanks for pointing this out, if it doesn't read that way at first glance it's better to change.

**Comment by oafischer** [comment]
Made some changes to remove the ambiguity, thanks for pointing that out @chenz4027! Just waiting for peer/writing review now.

File: applications/manage_apps_helm.adoc [hunk 2]
```diff
@@ -56,4 +56,11 @@ spec:
   pathname: https://kubernetes-charts.storage.googleapis.com/
 ----
 
-*Note:* To see REST API, use the link:../apis/api.adoc#apis[APIs].
\ No newline at end of file
+*Note:* To see REST API, use the link:../apis/api.adoc#apis[APIs].
```
Comment (cadawson) [inline applications/manage_apps_helm.adoc]: To see "the" REST API, "the REST APIs" or a specific REST API? It reads a little rough for me, and I'm not sure how to fix it.

Comment (oafischer) [reply applications/manage_apps_helm.adoc]: Not sure, I didn't change that part. Don't know why GitHub shows a change in this section.

**Review by cadawson — APPROVED** [review]
Looks good. A few minor changes and one request for a rewording. No need to send it back, unless you want feedback. Thanks!

**Pull request merged**